frog/imports/client/App/TopBar.js

```diff
@@ -96,10 +96,9 @@ class LogoutMenu extends React.Component<*, *> {
           <MenuItem
             onClick={() => {
               sessionStorage.removeItem('frog.sessionToken');
-              Meteor.logout(() => {
-                this.props.history.push('/');
-                window.notReady();
-              });
+              Meteor.logout();
+              this.props.history.push('/');
+              window.notReady();
```

Comment:
Nit: make sure this change is linked with the PR

Reply:
Updated with develop